crates/stencil/src/script.rs

```diff
@@ -265,6 +265,10 @@ pub struct ScriptStencil {
     pub fun_flags: FunctionFlags,
 
     pub lazy_function_enclosing_scope_index: Option<ScopeIndex>,
+
+    pub is_standalone_function: bool,
+    pub was_function_emitted: bool,
+    pub is_singleton_function: bool,
```

Comment:
nit: Copy comments from stencil code base, such that we would not have to do it when moving from C++ to Rust to use stencil in the future.